src/config-loader/config-loader.ts

```diff
@@ -147,34 +160,58 @@ export const setup = (app: Express.Application) => {
 				);
 			}
 
-			await Promise.all(
-				data.models.map(async (model) => {
-					if (
+			const modelPromises: Dictionary<Promise<void>> = _(data.models)
+				.filter(
+					(model): model is RequiredField<typeof model, 'apiRoot'> =>
 						(model.abstractSql != null || model.modelText != null) &&
-						model.apiRoot != null
-					) {
-						try {
-							await sbvrUtils.executeModel(
-								tx,
-								model as sbvrUtils.ExecutableModel,
+						model.apiRoot != null,
+				)
+				.keyBy((model) => model.apiRoot)
+				.mapValues(async (model: Model) => {
+					try {
+						// We use a dummy await here because we need the async boundary
+						await null;
```

Comment:
It just straight up breaks without it because in that case it all runs synchronously up to the `modelPromises[translateTo]` check and in a synchronous world `modelPromises` would not have been initialized yet and you'd get an error like `Uncaught ReferenceError: Cannot access 'modelPromises' before initialization`. Here's a test case you can try running:
```js
const _ = require('lodash');
const x = _({ a: 1, b: 2 })
	.mapValues(async (v, k) => {
		console.log(x, k, v, await x[k]);
		return v + 1;
	})
	.value();

Reply:
Okay I can see the necessity in this way of implementation.